files/zh-cn/web/events/event_handlers/index.md

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
Please do not create completely untranslated documents in this translation repository